Snapyr/Internal/SnapyrUtils.m

```diff
@@ -63,6 +63,11 @@ + (nullable NSURL *)getAPIHostURL
     return [SnapyrUtils getAPIHostURL:NO];
 }
 
++ (nullable NSString *)getWriteKey
+{
+    return [NSUserDefaults.standardUserDefaults stringForKey:@"snapyr_write_key"];
```

Comment:
(See comment above regarding `getGroupUserDefaults()`)

How/when will this key be set, so that it's available for reading back here?